spring-boot-admin/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# -----------------------------------------------------------------------------
+# Spring Boot Admin Server build
+# -----------------------------------------------------------------------------
+
+FROM openjdk:8u171-jdk-alpine as build
```

Comment:
We've migrated to using Liberica JDKs.

Sample image: `bellsoft/liberica-openjdk-alpine:8`

Reply:
done.